diagnostic.md

```diff
@@ -3,36 +3,40 @@
 What are three advantages of using Ember?
 
 ```sh
-# your answer here
+Embers CLI, Ember usually will use much less code and look more elegant than most other
+frameworks, URL support
 ```
 
 What is the templating system used in Ember? How it is related to
 handlebars?
 
 ```sh
-# your answer here
+Templates are backed with context in both Handlebars and Ember, in Emeber the
```

Comment:
Yes, in ember the controller or component provides the context for the template.
Also, ember provides some built in helpers, like `link-to`, that aren't available in regular handlebars.
1/1